src/main/java/uk/gov/hmcts/reform/professionalapi/controller/response/ContactInformationResponse.java

```diff
@@ -7,40 +7,44 @@
 import java.util.List;
 
 import lombok.Getter;
+import lombok.NoArgsConstructor;
 import uk.gov.hmcts.reform.professionalapi.domain.ContactInformation;
 
 @Getter
+@NoArgsConstructor
 public class ContactInformationResponse {
 
     @JsonProperty
-    private final String addressLine1;
+    private String addressLine1;
     @JsonProperty
-    private final String addressLine2;
+    private String addressLine2;
     @JsonProperty
-    private final String addressLine3;
+    private String addressLine3;
     @JsonProperty
-    private final String townCity;
+    private String townCity;
     @JsonProperty
-    private final String county;
+    private String county;
     @JsonProperty
-    private final String country;
+    private String country;
     @JsonProperty
-    private final String postCode;
+    private String postCode;
     @JsonProperty
-    private  List<DxAddressResponse> dxAddress;
+    private List<DxAddressResponse> dxAddress;
 
-    public ContactInformationResponse(ContactInformation contactInfo) {
+    public ContactInformationResponse(ContactInformation contactInfo, boolean isDxRequired) {
         this.addressLine1 = contactInfo.getAddressLine1();
         this.addressLine2 = contactInfo.getAddressLine2();
         this.addressLine3 = contactInfo.getAddressLine3();
         this.townCity = contactInfo.getTownCity();
         this.county = contactInfo.getCounty();
         this.country = contactInfo.getCountry();
         this.postCode = contactInfo.getPostCode();
-        this.dxAddress = contactInfo.getDxAddresses()
+        if (isDxRequired) {
+            this.dxAddress = contactInfo.getDxAddresses()
                     .stream()
-                    .map(dxAddres -> new DxAddressResponse(dxAddres))
+                    .map(DxAddressResponse::new)
                     .collect(toList());

```

Comment:
This might break the existing organisation's who has dx address since we are not sending the dxAddress field in the response so ask exui to test in the PR to make sure flow is not breaking. without testing with exui dont push the changes to master

Reply:
This issue has been addressed here: https://github.com/hmcts/rd-professional-api/pull/694/commits/7e3348faec52037d20fe55ee7312fe1fd8eb37b3